x-pack/plugins/synthetics/server/saved_objects/synthetics_monitor.ts

```diff
@@ -41,11 +39,69 @@ export const SYNTHETICS_MONITOR_ENCRYPTED_TYPE = {
      * payload on the `secrets` key. This ensures performant decryption. */
     ...secretKeys,
   ]),
-  attributesToExcludeFromAAD: new Set([
-    ConfigKey.ALERT_CONFIG,
-    ConfigKey.METADATA,
-    ConfigKey.MAX_ATTEMPTS,
-    ...legacyConfigKeys,
+  attributesToIncludeInAAD: new Set([
```

Comment:
Not all of these are utilized in the new test. I created a monitor from the UI and used the resulting object. Should we expand the test to include all AAD attributes?

Reply:
I've updated the test source object to include all AAD attributes.